### DISCUSSION TITLE
Dialogs | Submit on Enter on Input

### DISCUSSION THREAD

**Discussion by seikosantana** [AvaloniaCommunity/Material.Avalonia#380] [Ideas]
I've been using this library for a while and I'm enjoying it so far. Thanks a lot for such an amazing library.

I think it's nice if we could have Enter button triggering the button with `IsPositive` if every textfield entries are valid in a `TextFieldsDialog`.

Even better if we could choose if enter button triggers the `IsPositive` button, or sets the focus on next field, or specific field.

**Comment by SKProCH** [comment]
Hello, @seikosantana 

Thanks for feedback. Material.Dialog was developed by @appleneko2001, and seems like he is busy rn. Currently I doesn't have time to properly support/develop Material.Dialogs module. So I think it will remain just an idea until he returns. However, if you send a PR, I will be happy to review/merge it.
